Packs/FeedThreatVault/Integrations/FeedThreatVault/FeedThreatVault_description.md

```diff
@@ -0,0 +1,10 @@
+## Get Your API Key
+  1. Log in to the [Palo Alto Networks Customer Service Portal](https://support.paloaltonetworks.com/Support) with an account that has Super User privileges.
+  2. Select **Assets** and then **API Key Management**.
+  3. From the **Select an API key** drop down, select the API key based on the subscription type.
+  4. Set your API key in the API key field of the integration configuration
+   
+See [this documentation](https://support.paloaltonetworks.com/Support) for information about the API key.
```

Comment:
This is not a document.  
How about:
See [here}(https://support.paloaltonetworks.com/Support) for information about the API key.
(However, I am not sure what I should click once I get to this page.)